README.md

```diff
@@ -1,9 +1,58 @@
 # dccmonitor
 
-This package is intended to assist data curators to check the status of metadata and documentation files uploaded via the [dccvalidator](https://sage-bionetworks.github.io/dccvalidator/) shiny application.
-
-Currently, dccmonitor is a collection of functions to gather data of interest from files within a Synapse file view. The goal is to add a shiny app dashboard for viewing the information.
+This package is intended to assist data curators to check the status of metadata and documentation files uploaded via the [dccvalidator](https://sage-bionetworks.github.io/dccvalidator/) shiny application. dccmonitor is primarily a shiny application, with the ability to use functions for gathering validation information using the package functions only, if desired.
 
 ## Installation
 
-`devtools::install_github("Sage-Bionetworks/dccmonitor")`
\ No newline at end of file
+dccmonitor uses the reticulate package with the [Synapse Python Client](https://github.com/Sage-Bionetworks/synapsePythonClient). See the [reticulate documentation](https://rstudio.github.io/reticulate/#python-version) for more information on setting up reticulate to work with your local Python environment. Additionally, see the [Synapse Python Client](https://github.com/Sage-Bionetworks/synapsePythonClient) for installation instructions.
+
+dccmonitor can be installed via devtools:
+``` R
+devtools::install_github("Sage-Bionetworks/dccmonitor")
+```
+
+## Shiny Application
+
+Using the dccmonitor shiny application requires that the user have a Synapse account, and the user needs access to the specific Synapse folder of interest for monitoring (see Customization for details).
+
+After installing the package, the following will run the application locally:
+
+``` R
+library(dccmonitor)
+shiny::runApp()
```

Comment:
```suggestion
run_app()
```

`shiny::runApp()` only works if you're in a directory with an app.R file or server.R/ui.R, i.e. if you've cloned the repo. `run_app()` should work regardless.

Reply:
That's a good point. I didn't think of the differences between cloning and installing. I had originally put `run_app()`, but when I did it, I realized that the config setting is skipped (since it's not running from app.R) and changed it back to `shiny::runApp()`. I think I'll add instructions for both ways.

Reply:
If you want inspiration, @kelshmo has updated the language about this for dccvalidator here: https://github.com/Sage-Bionetworks/dccvalidator/pull/242

Reply:
Nice! I'll check it out.